app/models/interest.rb

```diff
@@ -0,0 +1,3 @@
+class Interest < ApplicationRecord
+  belongs_to :user
```

Comment:
Yes you have right but i try bulid thist app with nubmer of app requirement and in number 3 it wasn`t exactly say what AS sholud so i want to change this in #5 but i can change this now